package.json

```diff
@@ -240,5 +242,8 @@
     "tlds": "1.203.1",
     "xmlrpc": "1.3.2"
   },
-  "devDependencies": {}
```

Comment:
We can't have devDependencies in Volto, because we want them all installed on create-volto-app projects. So all should go to deps.

Reply:
Fixed.